src/libsync/vfs/cfapi/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+if (WIN32)
+    add_definitions(-D_WIN32_WINNT=_WIN32_WINNT_WIN10)
+
+    add_library("${synclib_NAME}_vfs_cfapi" SHARED
+        cfapiwrapper.cpp
+        hydrationjob.cpp
+        vfs_cfapi.cpp
+    )
+
+    target_link_libraries("${synclib_NAME}_vfs_cfapi"
+        "${synclib_NAME}"
+        cldapi
+    )
+
+    set_target_properties("${synclib_NAME}_vfs_cfapi" PROPERTIES
+        LIBRARY_OUTPUT_DIRECTORY ${BIN_OUTPUT_DIRECTORY}
+        RUNTIME_OUTPUT_DIRECTORY ${BIN_OUTPUT_DIRECTORY}
+        PREFIX ""
+        AUTOMOC TRUE
+    )
+
+    target_include_directories("${synclib_NAME}_vfs_cfapi" BEFORE PUBLIC ${CMAKE_CURRENT_BINARY_DIR} INTERFACE ${CMAKE_BINARY_DIR})
+
+    if(APPLE)
```

Comment:
@mgallien I am pretty sure there is a reason behind this, but, this part makes me confused. `APPLE` defined when `WIN32` is defined too along with the fact that it's inside `cfapi` folder that only makes sense for Windows.

Reply:
@mgallien Update: I guess this part is just a copy-paste from other CMakeLists.txt files and must be removed.